packages/js-drive/lib/abci/handlers/finalizeBlock/deliverTxFactory.js

```diff
@@ -282,10 +274,12 @@ function deliverTxHandlerFactory(
       `${stateTransition.constructor.name} execution took ${deliverTxTiming} seconds and cost ${actualStateTransitionFee} credits`,
```

Comment:
I think we need to revisit logger. DeliverTx is not a method any more. We probably need to create child logger in finalizeBlock and then pass it internal methods to extend it further. For method we could use something like "finalizeBlock#deliverTx"?